service/src/main/java/cz/muni/fi/pa165/facade/StoreFacadeImpl.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.muni.fi.pa165.facade;
+
+import cz.muni.fi.pa165.dto.StoreDTO;
+import cz.muni.fi.pa165.entity.Store;
+import java.util.List;
+
+import cz.muni.fi.pa165.service.BeanMappingService;
+import cz.muni.fi.pa165.service.StoreService;
+import javax.inject.Inject;
+
+/**
+ *
+ * @author rk
+ * @date 2016-11-23
+ */
+public class StoreFacadeImpl implements StoreFacade{
+    @Inject
+    private StoreService storeService;
+
+    @Inject
+    private BeanMappingService beanMappingService;
+    
+    @Override
+    public void createStore(StoreDTO store) {
+        storeService.createStore(beanMappingService.mapTo(store, Store.class));
+    }
+
+    @Override
+    public List<StoreDTO> findAll() {
+        return beanMappingService.mapTo(storeService.findAll(), StoreDTO.class);
```

Comment:
Uz len posledna vec, inak to vyzera dobre. Tieto metody, treba osetrit tak ako som to spravil ja tu: https://github.com/rudky4/PA165---Liquor-bottles-registry/pull/11/files